vars/Kali_GNU_Linux.yml

```diff
@@ -0,0 +1,8 @@
+---
+# vars file for Kali
+
+# The pip package names.  These values are identical to those fro
+# Debian 11.
+package_names:
+  - python3-pip
+  - python3-dev
```

Comment:
After the reply from @jsf9k I went to check out what the install would entail in a Docker instance of `kali-rolling`. When I installed the `python3-pip` package it also installed the `python-pip-whl` package. Which makes sense because as the [Debian package info page](https://packages.debian.org/unstable/python3-pip) shows, the `python-pip-whl` package is a dependency. Looks like I was a bit overzealous in my installation concerns.

Reply:
Although it spawned @mcdonnnj's insight, my comment was incorrect.  It appears that `python-pip-whl` is a python3 thang as well.

Reply:
***THANG!***